appservice/src/createAppService/AppKind.ts

```diff
@@ -10,7 +10,8 @@ export enum WebsiteOS {
 
 export enum AppKind {
     app = 'app',
-    functionapp = 'functionapp'
+    functionapp = 'functionapp',
+    workflowapp = 'functionapp,workflowapp'
```

Comment:
This should be just workflowapp since we can have more stuff here depending on the custom location picked up for LIMA